pkg/cmd/codespace/list.go

```diff
@@ -53,13 +67,14 @@ func newListCmd(app *App) *cobra.Command {
 	listCmd.Flags().StringVarP(&opts.userName, "user", "u", "", "The `username` to list codespaces for (used with --org)")
 	cmdutil.AddJSONFlags(listCmd, &exporter, api.CodespaceFields)
 
+	listCmd.Flags().BoolVarP(&opts.useWeb, "web", "w", false, "List codespaces in the web browser, can only be used with --repo flag")
```

Comment:
It might just be me, but this seems to read that `--repo` MUST be specified for `--web`. I think the error message like you used in `create` is easier to understand `cannot be used with --user or --org`

Reply:
fixed in [d167a17](https://github.com/cli/cli/pull/7288/commits/d167a1761bb271b3e05919ad9edf0a237adeb7d2)